packages/labs/react/src/create-component.ts

```diff
@@ -121,7 +156,10 @@ type EventProps<R extends Events> = {
  * messages. Default value is inferred from the name of custom element class
  * registered via `customElements.define`.
  */
-export const createComponent = <I extends HTMLElement, E extends Events = {}>(
+export const createComponent = <
+  I extends HTMLElement,
+  E extends EventNames = {}
+>(
```

Comment:
We should be using `ReactWebComponent<I, E>` as the return type here now.

Reply:
yessir!

Reply:
lol did everything but add a return type xD

Reply:
done!